ani.sadaghyan/repo/Lesson_3/exercise_18/exercise_18.py

```diff
@@ -0,0 +1,5 @@
+radius = int(input("Enter a radius of a circle: "))
+PI = 3.14159
+print("Circle's area is", round(PI * radius ** 2, 2))
```

Comment:
LW: Do not use round() method. 

Hint: Use manual calculation without a round(), where the radius is equal to 29.